guides/getting-started.mdx

```diff
@@ -115,8 +115,8 @@ You can now start using our components!
 
 ## What's next?
 
-[Explore our example site](https://steelthreads.github.io/agds-next/example-site/index.html) to see our list of interface examples, as well as the [related code on GitHub](https://github.com/steelthreads/agds-next/tree/main/example-site).
+[Explore our example site](/example-site) to see our list of interface examples, as well as the [related code on GitHub](https://github.com/steelthreads/agds-next/tree/main/example-site).
```

Comment:
Will this work in live? I've been burnt by the /agds-next path in our URL a few times

Reply:
Spoke about this verbally. All good 👍